src/github/create-commit.ts

```diff
@@ -34,15 +35,19 @@ export async function createCommit(
   treeSha: string,
   message: string
 ): Promise<string> {
-  const commitData = (
-    await octokit.git.createCommit({
+  try {
+    const {
+      data: {sha, url},
+    } = await octokit.git.createCommit({
       owner: origin.owner,
       repo: origin.repo,
       message,
       tree: treeSha,
       parents: [refHead],
-    })
-  ).data;
-  logger.info(`Successfully created commit. See commit at ${commitData.url}`);
-  return commitData.sha;
+    });
+    logger.info(`Successfully created commit. See commit at ${url}`);
+    return sha;
+  } catch (e) {
+    throw new CommitError(`Error creating commit for: ${treeSha}`, e as Error);
```

Comment:
It's annoying that TypeScript doesn't type the argument passed to `catch`, I wish this `as` could be avoided -- one option would be:

if (e instanceof Error) {
  throw new CommitError(...);
} else {
   throw e;
}